data/data/azure/master/master.tf

```diff
@@ -1,7 +1,7 @@
 locals {
   // The name of the masters' ipconfiguration is hardcoded to "pipconfig". It needs to match cluster-api
   // https://github.com/openshift/cluster-api-provider-azure/blob/master/pkg/cloud/azure/services/networkinterfaces/networkinterfaces.go#L131
-  ip_configuration_name = "pipconfig"
+  ip_configuration_name = "pipConfig"
```

Comment:
This probably needs to be separate commit if we want to revert this commit cleanly.

Reply:
moved